### PR TITLE
[Minor] Mention TCP port used by the playground docker containers

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Add a table of TCP ports used by the playground docker containers.

### Why are the changes needed?

Make it clearer to users that they might have issues if running local services on those ports.

Fix: # N/A

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Build locally.

### CODE REVIEW

**Comment by jerryshao** [comment]
@qqqttt123 please help to review it.

File: docs/how-to-use-the-playground.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ Depending on your network and computer, startup time may take 3-5 minutes. Once
 
 You first need to install git and docker-compose.
 
+## TCP ports used
+
+The playground runs a number of services. The TCP ports used may clash with existing services you run, such as MySQL or Postgres.
```
Comment (qqqttt123) [inline docs/how-to-use-the-playground.md:21]: `Postgres` ->  `PostgreSQL`?

Comment (justinmclean) [reply docs/how-to-use-the-playground.md:21]: Postgres is a commonly used name for PostgreSQL

Comment (qqqttt123) [reply docs/how-to-use-the-playground.md:21]: We use PostgreSQL in other places. Should we unify them?

Comment (justinmclean) [reply docs/how-to-use-the-playground.md:21]: Depending on the sentence it makes sense to use one or the other here "MySQL or PostgreSQL" is a little awkward/repetitive so I went with "MySQL or Postgres".

Comment (qqqttt123) [reply docs/how-to-use-the-playground.md:21]: OK.

**Review by qqqttt123 — APPROVED** [review]
LGTM.

**Pull request merged**